docs/concepts/overview/working-with-objects/common-labels-annotations.md

```diff
@@ -0,0 +1,186 @@
+applications but are not required for any core tooling to work.
+
+One common theme you'll notice about the labels and annotations is that they
+contain a prefix of `app.kubernetes.io`. Labels and annotations without a prefix
+are considered private to users. These are meant to be common labels that do not
+interfere with existing or custom user labels. To accomplish this a prefix is
+used.
+
+## Labels
+
+| Key                                 | Description           | Example  | Type |
+| ----------------------------------- | --------------------- | -------- | ---- |
+| `app.kubernetes.io/name`              | The name of the application stack | `wordpress` | string |
+| `app.kubernetes.io/version`           | The current version of the running application (e.g., a semantic version, revision hash, etc.) | `v1.2.3` | string |
+| `app.kubernetes.io/component`         | The component within the architecture | `front-end` | string |
+| `app.kubernetes.io/component-name`    | The name of a component within the application | `mysql` | string |
+| `app.kubernetes.io/component-version` | The current version of the component | `5.7.21` | string |
+| `app.kubernetes.io/deploy-manager`    | The tool being used to manage the operation of an application | `helm` | string |
+
+To illustrative these labels in action consider the following StatefulSet object
+containing them:
+
+```yaml
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/name: wordpress
+    app.kubernetes.io/version: "4.9.4"
+    app.kubernetes.io/component: database
+    app.kubernetes.io/component-name: mysql
+    app.kubernetes.io/component-version: "5.7.21"
+    app.kubernetes.io/deploy-manager: helm
+```
+
+## Annotations
+
+| Key                     | Description           | Example  | Type |
+| ----------------------- | --------------------- | -------- | ---- |
+| `app.kubernetes.io/usage` | The location of an applications usage information (e.g., details for how it's being run by an organization). A preference is given to version specific information | `https://example.com/foo/1.2/` | URL |
+| `app.kubernetes.io/website`   | A website to find out more information about the application in general | `https://wordpress.org` | URL |
+
+To illustrative these annotations in action consider the following StatefulSet object
+containing them:
+
+```yaml
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  annotations:
+    app.kubernetes.io/usage: https://github.com/kubernetes/charts/tree/master/stable/mysql
+    app.kubernetes.io/website: https://www.mysql.com/
+```
+
+## Examples
+
+To illustrate different ways to use these labels the following examples have varying complexity.
+
+### A Simple Stateless Service
+
+Consider the case for a simple stateless service deployed using `Deployment` and `Service` objects. The following two snippets represent how the labels could be used in their simplest form.
```

Comment:
Would be helpful to elaborate on how these are useful.  e.g. These could then be displayed in a dashboard as a single entry composed of the Deployment and Service.